packages/common/src/store/buy-usdc/utils.ts

```diff
@@ -9,26 +15,101 @@ import {
   MIN_USDC_PURCHASE_AMOUNT_CENTS,
   BUY_TOKEN_VIA_SOL_SLIPPAGE_BPS
 } from '~/services/remote-config/defaults'
-import { getContext } from '~/store/effects'
-import { getFeePayer } from '~/store/solana/selectors'
+
+import { getContext } from '../effects'
+import { getFeePayer } from '../solana/selectors'
+
+const POLL_ACCOUNT_INFO_DELAY_MS = 1000
+const POLL_ACCOUNT_INFO_RETRIES = 30
```

Comment:
just curious on if there's a reason for this number from a Solana standpoint?

Reply:
Not specifically. I was being a bit arbitrary here. But 30 seconds should be more than sufficient for a recently committed transaction to be indexed by RPCs.